internal/cliwrapper/cliwrapper.go

```diff
@@ -53,6 +53,19 @@ func (p *cliWrapper) ImageExists(image string) (*bool, error) {
 	return &exists, nil
 }
 
+func (p *cliWrapper) ContainerExists(containerID string) (bool, error) {
```

Comment:
Technically, this shows *running* containers, whereas `--all` will *also* show terminated containers that "exist".